### DISCUSSION TITLE
[img2img] ControlNet: Resize Mode greyed out

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#1603] [General]
Hi, see topic. Is this related to a1111 or some controlnet plugin? I have no idea what broke this, but currently I cannot change the resize options anymore (Resize Mode), as they are greyed out.

![screenshot](https://i.gyazo.com/122239f4bcf39cbcedaef0adf675ad52.png)

I am trying to do an "inpaint with lama".

Also the other problem with inpaint with lama still persists: https://github.com/vladmandic/automatic/discussions/1571

**Comment by huchenlei** [comment]
Resize mode is greyed out as if you don’t provide controlnet image, resize mode from a1111 will be used. I.e. even you can select, the value won’t be used.

You can either upload an image to controlnet and select resize mode in controlnet UI or select resize mode in a1111 img2img UI

**Reply by huchenlei** [reply]
If you hover on the resize mode radio, there is a hover text explaining that

**Reply by Tillerz** [reply]
Ok, thank you. :)
